src/v/pandaproxy/rest/configuration.cc

```diff
@@ -54,6 +56,24 @@ configuration::configuration()
       "consumer_instance_timeout_ms",
       "How long to wait for an idle consumer before removing it",
       {},
-      std::chrono::minutes{5}) {}
-
+      std::chrono::minutes{5})
+  , client_cache_max_size(
+      *this,
+      "client_cache_max_size",
+      "The maximum number of kafka clients in the LRU cache",
+      {.needs_restart = config::needs_restart::no},
+      10)
```

Comment:
Should this validate that the cache isn't empty?

Reply:
Yes. good point.